src/main/java/GameEngine.java

```diff
@@ -203,10 +201,10 @@ private void sleep(int ms) {
     private void namePlayers() throws SQLException, ClassNotFoundException {
         System.out.println("Enter name for player one:");
         player1.setName(inputProcessor.nextLine());
-        highScoreDB.addUserIfNew(player1.getName());
+   /*     highScoreDB.addUserIfNew(player1.getName());*/
```

Comment:
är det utkommenterat pga mysql skäl? =)

Reply:
ok! hade mergat in det men nu är det konflikt!

Reply:
fixxed